### PR TITLE
Homework_4

### PR DESCRIPTION
**Pull request by VadimMir**

### CODE REVIEW

File: homeworks/VadimMiroshnik_VadimMir/homework_4/homework4_2.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+function card(){
+    let card = '4561261212345467';
+   
+    let  sum_unpaired = 0, paired_Elemen = 0, unpaired_Elemen = 0, sum_paired = 0, sum = 0, sum_Card;
+    arrayCard = card.match(/[\S\s]{1,1}/g);
```
Comment (smithua) [inline homeworks/VadimMiroshnik_VadimMir/homework_4/homework4_2.js]: format your code, it is hard to read

Comment (VadimMir) [reply homeworks/VadimMiroshnik_VadimMir/homework_4/homework4_2.js]: First and second task?

Comment (VadimMir) [reply homeworks/VadimMiroshnik_VadimMir/homework_4/homework4_2.js]: added comments, if in my case this is correct)

**Comment by smithua** [comment]
Папка .vscode должна быть добавлена в глобальный .gitignore
https://help.github.com/articles/ignoring-files/

**Comment by VadimMir** [comment]
Добавил

**Pull request closed without merging**